templates/main/login.html

```diff
@@ -7,6 +7,29 @@
         <script src="js/login.js" defer></script>
     </head>
     <body>
+    	<div id='container'>
         <h1>Login page</h1>
+        <form action="" method="POST">
+			  
+			  	
+			    <label for="uname"><b>Username</b></label>
+			    <input type="text" placeholder="Enter Username" name="uname" required>
+			    <br>
+			    <label for="psw"><b>Password</b></label>
+			    <input type="password" placeholder="Enter Password" name="psw" required>
+			    <br>
+			    <button type="submit" name='submitbutton'>Login</button>
+			    <br>
+			    <br>
+			    <h10>Don't have an account? <a href="signup.php">Sign up</a></h10>
+		</form> 
+                
+		<?php if (empty($params["msg"])): ?>
+		
+       <?php else: ?>
```

Comment:
An empty `if` with an `else` statement means that we can negate the condition in the `if` and remove the `else`, right?